mpmath/function_docs.py

```diff
@@ -2291,6 +2291,10 @@
 
 For larger `n`, `B_n` is evaluated in terms of the Riemann zeta
 function.
+
+.. versionchanged :: 1.4
+    `bernoulli(1)` gives `+0.5` instead of `-0.5`. The previous
+    behavior can be obtained with `(-1)**n*bernoulli(n)`.
```

Comment:
I think the used convention should be also documented in the description of the function, before "examples" section.  And for the bernfrac() as well.  Also keep in mind that the default_role of sphinx is "math" for the mpmath.  So, for code you should use double backticks.

Reply:
Done.